packages/blockchain-wallet-v4-frontend/src/scenes/BuySell/SfoxCheckout/Content/JumioStatus/template.success.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react'
+import styled from 'styled-components'
+import { FormattedMessage } from 'react-intl'
+import { Button, Text } from 'blockchain-info-components'
+
+const Container = styled.div`
+  padding: 15px;
+  border: 1px solid ${props => props.theme['gray-1']};
+`
+const Title = styled(Text)`
+  color: ${props => props.theme['brand-primary']};
+  margin-bottom: 10px;
+`
+const Body = styled(Text)`
+  font-size: 14px;
+  font-weight: 300;
+`
+const ButtonWrapper = styled.div`
+  display: flex;
+  justify-content: center;
+  margin-top: 10px;
+`
+
+export const Success = ({ value, onClick }) => {
+  const { status } = value
+  const statusTitleHelper = status => {
+    switch (status) {
+      case 'PENDING':
```

Comment:
`status === 'PENDING' &&` is shorter than `switch`

Reply:
I will need to add more cases to the switch in the near future

Reply:
See WEB4-1167